plugins/parsers/xml/README.md

```diff
@@ -53,7 +53,8 @@ that need to be specified in a `fields_int` section.
 ```
 
 A configuration can contain muliple *xml* subsections for e.g. the file plugin to process the xml-string multiple times.
-Consult the [XPath syntax][xpath] and the [underlying library's functions][xpath lib] for details and help regarding XPath queries.
+Consult the [XPath syntax][xpath] and the [underlying library's functions][xpath lib] for details and help regarding XPath queries. Consider using an XPath tester such as [xpather.com](http://xpather.com/) or [Code Beautify's XPath Tester](https://codebeautify.org/Xpath-Tester) for help developing and debugging 
```

Comment:
Could you keep the same link syntax as the rest of the document and add the link references at the end of the document?